src/connection.ts

```diff
@@ -912,6 +912,8 @@ class Connection extends EventEmitter {
         }
 
         this.config.options.trustServerCertificate = config.options.trustServerCertificate;
+      } else {
+        deprecate('The default value for `config.options.trustServerCertificate` will change from `true` to `false` in the next major version of `tedious`. Set the value to `true` or `false` explicitly to silence this message.');
```

Comment:
Hi @arthurschreiber. I just had a question. Is this change only a message for users to get aware of the change ahead? The actual default value change for line 537 will be in another PR?

Reply:
Yes exactly. If we change the default, a lot of applications that rely on `trustServerCertificate` defaulting to `true` will break, so I'd like to do that change together with a major version bump.

By emitting a value when `trustServerCertificate` is not set (and not emitting the message if a value is provided), we can nudge users to set a value explicitly. If we then change the default, they will be unaffected (because they provided an explicit value).